package.json

```diff
@@ -154,14 +154,20 @@
                     "type": "boolean",
                     "default": false,
                     "description": "Toggle italic/bold without selecting text"
+                },
+                "markdown.extension.showExplorer": {
```

Comment:
I think making this config named `showOutline` or `showOutlineInExplorer` will be better. What is your opinion?

Reply:
@neilsustc I took `showExplorer` name from Microsoft's docker extension. Might be wise to match them and keep things consistent.

https://github.com/Microsoft/vscode-docker/blob/master/package.json#L235-L238

Reply:
Alright. Thanks very much ❤